tests/utils/dockercli/volumelifecycle.go

```diff
@@ -30,15 +30,26 @@ import (
 // CreateVolume is going to create vsphere docker volume with given name.
 func CreateVolume(ip, name string) (string, error) {
 	log.Printf("Creating volume [%s] on VM [%s]\n", name, ip)
-	return ssh.InvokeCommand(ip, dockercli.CreateVolume+"--name="+name)
+	return ssh.InvokeCommand(ip, dockercli.CreateVolume + " --name= " + name) 
```

Comment:
I agree .. for some of us `Visual studio code's gofmt plugin` auto formats hence the comment .. anyway it's nit.